packages/web3-eth-ens/src/config.ts

```diff
@@ -36,6 +42,9 @@ export const methodsInInterface: { [T: string]: string } = {
 	contenthash: 'contenthash',
 };
 
+/**
+ * An object holding the addressed of the ENS registries on the different networks (mainnet, ropsten, rinkeby, goerli).
+ */
 export const registryAddresses: { [T: string]: string } = {
 	main: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
 	ropsten: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
```

Comment:
Its not in scope of this issue but some of these networks (like ropsten, and rinkeby  ) are deprecated could you update this list?